components/TextFields/src/MDCTextInputControllerOutlined.m

```diff
@@ -60,13 +60,13 @@ - (void)setFloatingEnabled:(__unused BOOL)floatingEnabled {
   // Unused. Floating is always enabled.
 }
 
-- (CGPoint)floatingPlaceholderDestination {
-  CGPoint destination = [super floatingPlaceholderDestination];
+- (UIOffset)floatingPlaceholderOffset {
+  UIOffset destination = [super floatingPlaceholderOffset];
   CGFloat offset = self.textInput.placeholderLabel.font.lineHeight -
                    self.textInput.placeholderLabel.font.xHeight;
-  destination.y = -1 * offset;
+  destination.vertical = -1 * offset;
   MDCPaddedLabel *placeholderLabel = (MDCPaddedLabel *)self.textInput.placeholderLabel;
-  destination.x -= placeholderLabel.horizontalPadding;
+  destination.horizontal += placeholderLabel.horizontalPadding;
```

Comment:
Is this operand change (`+` &rarr; `-`) a bug fix or the result of moving from a destination to an offset?

Reply:
Result of change to offset

Reply:
Ack.